docker/mingw.sh

```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/env bash
 
-set -x
+set -xe
```

Comment:
Make this `set -euo pipefail` like in all other scripts.

Reply:
Fixed. Thanks.